docs/getting_started/solo/software_setup.rst

```diff
@@ -0,0 +1,237 @@
+
+ALOHA Software Installation
+===========================
+
+1.  Clone the Interbotix fork of ALOHA into the workspace's source directory:
+
+  .. code-block:: bash
+
+    $ cd ~/interbotix_ws/src
+    $ git clone https://github.com/Interbotix/aloha.git -b 2.0
+
+2.  Run rosdep to install any dependencies:
+
+  .. code-block:: bash
+
+    $ cd ~/interbotix_ws
+    $ rosdep install --from-paths src --ignore-src -r -y
+
+3.  Set the ``InterbotixManipulatorXS``'s ``iterative_update_fk`` default value to ``False`` at ``~/interbotix_ws/src/interbotix_ros_toolboxes/interbotix_xs_toolbox/interbotix_xs_modules/interbotix_xs_modules/xs_robot/arm.py`` (`link`_).
+
+.. _`link`: https://github.com/Interbotix/interbotix_ros_toolboxes/blob/c187bcea89b60391244bb19943ebd78f770aa975/interbotix_xs_toolbox/interbotix_xs_modules/interbotix_xs_modules/xs_robot/arm.py#L81
+
+4.  Build the workspace:
+
+  .. code-block:: bash
+
+    $ cd ~/interbotix_ws
+    $ colcon build
+
+.. note::
+
+  If planning to change the control or data collection software later on, you may want to do a symbolically-linked install.
+  If that is the case, remove the build and install directories, and re-run ``colcon build`` with the ``--symlink-install`` flag.
+
+    .. code-block:: bash
+
+      $ cd ~/interbotix_ws
+      $ rm -rf build install
+      $ colcon build --symlink-install
+
+Post-Install Hardware Setup
+===========================
+
+The arm and cameras need to be bound to a unique device.
+The following sections will provide steps on setting up unique symbolic links for each device.
+
+Arm Symlink Setup
+-----------------
+
+We will configure udev rules to bind the arms to specific device names. 
+Depending on the orientation of the pair you plan to use, configure the device names accordingly as either left or right:
+
+* ``ttyDXL_leader_left`` for the left oriented leader arm
+* ``ttyDXL_follower_left`` for the follower arm
+* ``ttyDXL_leader_right`` for the right oriented leader arm
+* ``ttyDXL_follower_right`` for the follower arm
+
+
+To set these up, do the following:
+
+1.  Plug in only the leader left robot to the computer.
+
+2.  Determine its device name by checking the ``/dev`` directory before and after plugging the device in.
+    This is likely something like ``/dev/ttyUSB0``.
+
+3.  Print out the device serial number by running the following command:
+
+  .. code-block:: bash
+
+    $ udevadm info --name=/dev/ttyUSB0 --attribute-walk | grep ATTRS{serial} | head -n 1 | cut -d '"' -f2
+    FT88YWBJ
+
+4.  The output of the command will look like ``FT88YWBJ`` and be the serial number of the arm's U2D2 serial converter.
+
+5.  Add the following line to the computer's fixed Interbotix udev rules at ``/etc/udev/rules.d/99-fixed-interbotix-udev.rules``:
+
+  .. code-block:: bash
+
+    SUBSYSTEM=="tty", ATTRS{serial}=="<SERIAL NUMBER>", ENV{ID_MM_DEVICE_IGNORE}="1", ATTR{device/latency_timer}="1", SYMLINK+="ttyDXL_leader_left"
```

Comment:
> ttyDXL_leader_left

Should this change based on their orientation or does this not matter?

Reply:
For logical consistency, if you are using a leader on the right, the follower should also be configured as right.